freeipa/src/main/resources/freeipa-salt/salt/nginx/conf/ssl-locations.d/nodestatus.conf

```diff
@@ -1,5 +1,5 @@
 location /nodestatus {
-  proxy_pass         https://{{ salt['grains.get']('fqdn') }}:61888;
+  proxy_pass         http://127.0.0.1:61888;
```

Comment:
Isn't is still supposed to be https?

Reply:
not yet, it will be similar as satboot, basi auth without ssl (actually won't be anything secret in the communication, maybe later ssl can be added if needed)